vspace/src/main/java/edu/asu/diging/vspace/web/staff/DefaultImageController.java

```diff
@@ -0,0 +1,89 @@
+import edu.asu.diging.vspace.core.services.IImageService;
+
+@Controller
+public class DefaultImageController {
+
+    public static final String API_DEFAULT_SPACE_IMAGE_PATH = "/api/image/default/link/space/";
+    public static final String API_DEFAULT_MODULEIMAGE_PATH = "/api/image/default/link/module/";
+    public static final String API_DEFAULT_EXTERNALIMAGE_PATH = "/api/image/default/link/external/";
+    public static final String API_DEFAULT_SPACE_IMAGE_STATUS = "/api/exhibition/default/link/image/status/";
+
+    @Autowired
+    private IImageService imageService;
+
+    @Autowired
+    private IExhibitionManager exhibitManager;
+
+    @RequestMapping(value = API_DEFAULT_SPACE_IMAGE_PATH, method = RequestMethod.GET)
+    public ResponseEntity<byte[]> getDefaultSpaceImage() {
+        IExhibition exhibition = exhibitManager.getStartExhibition();
+        IVSImage spaceImage = exhibition.getSpaceLinkDefaultImage();
+        return getResponseWithDefaultHeaders(spaceImage);
+    }
+
+    @RequestMapping(value = API_DEFAULT_MODULEIMAGE_PATH, method = RequestMethod.GET)
+    public ResponseEntity<byte[]> getDefaultModuleImage() {
+        IExhibition exhibition = exhibitManager.getStartExhibition();
+        IVSImage moduleImage = exhibition.getModuleLinkDefaultImage();
+        return getResponseWithDefaultHeaders(moduleImage);
+    }
+
+    @RequestMapping(value = API_DEFAULT_EXTERNALIMAGE_PATH, method = RequestMethod.GET)
+    public ResponseEntity<byte[]> getDefaultExternalImage() {
+        IExhibition exhibition = exhibitManager.getStartExhibition();
+        IVSImage externalLinkImage = exhibition.getExternalLinkDefaultImage();
+        return getResponseWithDefaultHeaders(externalLinkImage);
+    }
+
+    @RequestMapping(value = API_DEFAULT_SPACE_IMAGE_STATUS, method = RequestMethod.GET)
+    public ResponseEntity<String> getDefaultImageStatus() {
+        IExhibition exhibition = exhibitManager.getStartExhibition();
+        IVSImage defaultSpaceImage = exhibition.getSpaceLinkDefaultImage();
+        IVSImage defaultModuleImage = exhibition.getModuleLinkDefaultImage();
+        IVSImage defaultExternalLinkImage = exhibition.getExternalLinkDefaultImage();
+
+        boolean defaultSpaceImageFlag = defaultSpaceImage != null? true : false;
+        boolean defaultModuleImageFlag = defaultModuleImage != null? true : false;
+        boolean defaultExternalLinkImageFlag = defaultExternalLinkImage != null? true : false;
+
+        JsonObject jsonObj = new JsonObject();
+        jsonObj.addProperty("defaultSpaceImageFlag", defaultSpaceImageFlag);
+        jsonObj.addProperty("defaultModuleImageFlag", defaultModuleImageFlag);
+        jsonObj.addProperty("defaultExternalLinkImageFlag", defaultExternalLinkImageFlag);
+        return new ResponseEntity<>(jsonObj.toString(), HttpStatus.OK);
+    }
+
+    private ResponseEntity<byte[]> getResponseWithDefaultHeaders(IVSImage image) {
+        if(image == null) {
+            return null;
```

Comment:
I don't think this should return null, this should return a response entity that the client understands (some kind of error code)